src/js/hyperschedule.ts

```diff
@@ -527,9 +527,51 @@ function coursePassesTextFilters(
   return true;
 }
 
-function parseDaysInequality(inputDays: string) {
-  for (const rel of filterInequalities)
-    if (inputDays.startsWith(rel)) return rel;
+// timeFilters is a one or two element array
+// [start_time] or [start_time, end_time]
+function coursePassesTimeFilters(
+  course: Course.CourseV3,
+  timeFilters: string[]
+) {
+  // indicates no current time filters
+  if (timeFilters[0] === "") {
+    return true;
```

Comment:
Is there a way to filter on _just the end time_/is that something you should be able to do?  If so, maybe this check should also consider `timeFilters[1]`.

Reply:
Right now, there is no way to filter on the end time. With the way the regex is set up now, there isn't a way for there to be a `timeFilters[1]` if there isn't a `timeFilters[0]`. I personally think filtering on end time might be a bit weird.